app/grandchallenge/core/static/css/core.css

```diff
@@ -242,3 +242,10 @@ a[aria-expanded="false"] .fa-chevron-down {
 .htmx-request.htmx-indicator {
     display: inline-block;
 }
+
+blockquote {
+    padding: 0 1em;
+    font-size: inherit !important;
+    border-left: .25em solid var(--dark);
+    color: #595959;
```

Comment:
Could also use `var(--primary)` here, but opted for a lighter gray to mute the text a bit more, while still having enough contrast for accessibility (according to this website: https://webaim.org/resources/contrastchecker/)

I only found the colors defined in bootstrap. We don't have more colors defined somewhere do we?

Reply:
Is it possible to do this in base.scss and use the bootstrap spacing and colour variables? I think that is important for consistency.

Reply:
Yes, thank you for pointing me in that direction.